### PR TITLE
-remove custom-roof-coverage parameter

### PR DESCRIPTION
**Pull request by lguilhermers**
This solves #3179.
A boolean parameter that was reduntant was removed.

How to review:
1. check the code changes
2. Open on the dashboard (energy potantials -> photovoltaic panels / photovoltaic-thermal potantials)
3. Test simulation with different values for roof-coverage and confirm there are changes.

### CODE REVIEW

**Comment by martin-mosteiro** [comment]
Ok, I just did the following:
- Ran PV potential on the dashboard with 100% and 25% coverage
- Ran PVT potential on the dashboard with 100% and 25% coverage
As expected, the results are 75% lower in the second case, so the PR does what it's intended to do.

As mentioned during our meeting, I believe the rationale for the Boolean back then was to have a foolproof way to make sure users don't change the value in one simulation and then forget it's on on the next one. I agree that this is unnecessary. However, I do think without the Boolean toggle in the dashboard the roof coverage gets a bit lost in the tilt angle section of the tool. So I would suggest changing the order of the parameters on the dashboard as follows:
1. Annual threshold
2. Roof coverage
3. Tilt angle
That way it's 100% clear that the coverage part does not depend on the tilt angle boolean above it.

**Comment by martin-mosteiro** [comment]
@lguilhermers If implemented my suggestion, please check it out :)

**Comment by lguilhermers** [comment]
Hi Martin, it makes sense. I agree with the order as well.
I ran a test, but if you are able to do a final check, it would be great. Thanks!

**Comment by lguilhermers** [comment]
One minor thing I am sliding into this PR: the workflow quick wasn't retrieving from the reference scenario, but instead just running with the current scenario. Luckily I have a backup for it haha
I changed to be consistent with the other tests.

**Comment by martin-mosteiro** [comment]
@lguilhermers Looks good! Once the tests pass, I'll merge

**Review by martin-mosteiro — APPROVED** [review]

File: cea/default.config [hunk 1]
```diff
@@ -417,14 +421,6 @@ panel-tilt-angle = 10
 panel-tilt-angle.type = RealParameter
 panel-tilt-angle.help = Solar panel tilt angle if using user-defined tilt angle.
 
-custom-roof-coverage = False
-custom-roof-coverage.type = BooleanParameter
-custom-roof-coverage.help = Calculate solar panel potential while limiting the maximum coverage for any given roof surface to a user-defined limit.
-
-max-roof-coverage = 1.0
```
[the remark this answers is not in the document]
Comment (lguilhermers) [reply cea/default.config:424]: humm.. not sure, I was thinking I reverted those after your first comment. But I saw you changed back, committed and merged. Thanks for it!

[the remark this answers is not in the document]
Comment (martin-mosteiro) [reply cea/default.config:424]: Sorry, those comments are really old. Not sure why they appear as being from yesterday. Anyway, those comments were already resolved back then.

**Pull request merged**